t/ui/15-comments.t

```diff
@@ -176,7 +176,7 @@ subtest 'URL auto-replace' => sub {
         bsc#1234 boo#2345,poo#3456 t#4567
         t#5678/modules/welcome/steps/1
         https://progress.opensuse.org/issues/6789
-        https://bugzilla.novell.com/show_bug.cgi?id=1234
+        https://bugzilla.suse.com/show_bug.cgi?id=1234
```

Comment:
Do you agree with this approach which means that `bugzilla.novell.com` is *not* replaced automatically anymore and also not recorded as a bugref?